public/javascripts/notebook/observable.js

```diff
@@ -113,31 +113,48 @@ return new function () {
 
 
 
-  this.makeObservableHelper = function (id, kind, initialValue) {
+  this.makeObservableHelper = function (id, kind, initialValue, df) {
```

Comment:
this is rather complex now. maybe add a comment of what's happening here, something like
- until observable is not initialized we create a magic fn which:
  * simulates a delayed observable  by delaying `.subscribe` calls via a `$.Deferred`
  * passes over the `publishValue` to observable (?) once it's available
- .... (something else about lazy observable initialization)